src/test/java/org/folio/rest/impl/TemplateRequestTest.java

```diff
@@ -96,7 +93,6 @@ public static void setUpClass(final TestContext context) throws Exception {
         PostgresClient.setConfigFilePath(postgresConfigPath);
         break;
       case "embedded":
-        PostgresClient.setIsEmbedded(true);
```

Comment:
The database configuration in TemplateRequestTest.java is different from that in TemplateDeletionTest.java and RestVerticleTest.java and should be changed, for example by removing it and using the default configuration provided by PostgresClient: https://github.com/folio-org/raml-module-builder#postgresql-integration

Reply:
database configuration was changed